### PR TITLE
Enhancements

### PR DESCRIPTION
**Pull request by OskarStark**
WDYT @smoench ?

### CODE REVIEW

File: src/Command/AnalyzeCommand.php [hunk 1]
```diff
@@ -133,19 +133,18 @@ protected function execute(InputInterface $input, OutputInterface $output): int
...
-            $errorFormat = 'console';
 
             try {
                 $ci = $ciDetector->detect();
                 if (CiDetector::CI_GITHUB_ACTIONS === $ci->getCiName()) {
                     $errorFormat = 'github';
                 }
             } catch (CiNotDetectedException $e) {
-                // pass and use default
+                $errorFormat = 'console';
```
Comment (OskarStark) [inline src/Command/AnalyzeCommand.php:147]: WDYT @smoench ?

Comment (smoench) [reply src/Command/AnalyzeCommand.php:147]: Wouldn't work in an other CI environment. The CiDetector can detect more than github actions ;-)

Comment (OskarStark) [reply src/Command/AnalyzeCommand.php:147]: Makes sense

**Review by smoench — CHANGES_REQUESTED** [review]
I'm fine with the `detect`-keyword but the default error format handling needs to be reverted.

**Comment by OskarStark** [comment]
> the default error format handling needs to be reverted.

done

**Comment by smoench** [comment]
Can you update the readme adding a hint about the auto detection?

**Comment by OskarStark** [comment]
> Can you update the readme adding a hint about the auto detection?

Done, please review

File: README.md [hunk 2]
```diff
@@ -41,9 +41,13 @@ If your `*.rst` files are not located in root:
...
 
-DOCtor-RST has two error formatters `console` *(default)* and `github`. 
+* **detect** _(default)_ If running inside GithubActions CI environment, `github` is used, otherwise `console`.
 
-`github` is useful in the context of GitHub actions and can be activated with the option `--error-formatter=github`.
+* **github** Used to annotate your pull requests.
+
+* **console** Used as to generate a human readable output.
+
+To force the usage of a formatter, use the `--error-formatter` option.
```
Comment (OskarStark) [inline README.md:50]:
```suggestion
To force the usage of a specific formatter, use the `--error-formatter` option.
```

Comment (smoench) [inline README.md]:
```suggestion
To force the usage of a formatter, use the `--error-format` option.
```

File: README.md [hunk 3]
```diff
@@ -41,9 +41,13 @@ If your `*.rst` files are not located in root:
...
 
-DOCtor-RST has two error formatters `console` *(default)* and `github`. 
+* **detect** _(default)_ If running inside GithubActions CI environment, `github` is used, otherwise `console`.
 
-`github` is useful in the context of GitHub actions and can be activated with the option `--error-formatter=github`.
+* **github** Used to annotate your pull requests.
+
+* **console** Used as to generate a human readable output.
+
+To force the usage of a specific formatter, use the `--error-formatter` option.
```
Comment (OskarStark) [inline README.md:50]:
```suggestion
To force the usage of a specific formatter, use the `--error-format` option.
```

Comment (smoench) [inline README.md:50]:
```suggestion
To force the usage of a specific formatter, use the `--error-format` option.
```

Comment (OskarStark) [reply README.md]: Thanks!

**Pull request merged**